File: ios/ExpoTestFlightModule.swift
import ExpoModulesCore

public class ExpoTestFlightModule: Module {
  public func definition() -> ModuleDefinition {
    Name("ExpoTestFlight")

    Constants {
      let isTestFlight: Bool

#if DEBUG || targetEnvironment(simulator)
      isTestFlight = false
#else
      isTestFlight = Bundle.main.appStoreReceiptURL?.lastPathComponent == "sandboxReceipt"
#endif

      return [
        "isTestFlight": isTestFlight
      ]
    }
  }
}

File: android/src/main/java/expo/modules/testflight/ExpoTestFlightModule.kt
package expo.modules.testflight

import expo.modules.kotlin.modules.Module
import expo.modules.kotlin.modules.ModuleDefinition

class ExpoTestFlightModule : Module() {
  override fun definition() = ModuleDefinition {
    Name("ExpoTestFlight")

    Constants(
      "isTestFlight" to false
    )
  }
}
